tsconfig.json

```diff
@@ -12,10 +12,11 @@
         "resolveJsonModule": true,
         "isolatedModules": true,
         "noEmit": true,
-        "jsx": "preserve",
-        "noUnusedParameters": false,
-        "noUnusedLocals": false,
+        "jsx": "react",
+        "noUnusedParameters": true,
+        "noUnusedLocals": true,
```

Comment:
Personally, I prefer to leave `noUnusedXyz` checks to the linter. It's usually a nuisance on development. What do you think?

Reply:
Yes, we already have them on eslint config file, so they're redundant by the compiler and linter.